pkg/controllers/data.go

```diff
@@ -0,0 +1,146 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	. "github.com/Igoraamc/knot-cloud-storage/pkg/entities"
+	. "github.com/Igoraamc/knot-cloud-storage/pkg/interactor"
+	"github.com/gorilla/mux"
+)
+
+type DataController struct{}
+
+type DataInterface interface {
+	GetAll()
+	GetByID()
+	Create()
+}
+
+type errorMessage struct {
+	error   bool
+	message string
+}
+
+var dao = DataDAO{}
```

Comment:
This looks good, just make sure the entire file (not just the selected lines) has appropriate [function grouping and ordering](https://docs.google.com/document/d/1w61-0sp_QkuaCI3qepc9yxh0AEkCBuS0bOyKSahsOkQ/edit#heading=h.fwxbbk14050i) :confetti_ball: 

Let us know if you got any suggestions, tho :smile: 

```suggestion
var dao = DataDAO{}

type DataInterface interface {
	GetAll()
	GetByID()
	Create()
}

type DataController struct{}

type errorMessage struct {
	error   bool
	message string
}
```

Reply:
Done!